lib/shopify_api/rest/request.ex

```diff
@@ -197,6 +197,11 @@ defmodule ShopifyAPI.REST.Request do
 
   defp remaining_calls(_), do: nil
 
+  # Absolute URL generator
+  defp url(%{shop_name: domain}, <<?/, path::binary>>),
+    do: "#{ShopifyAPI.transport()}#{domain}/#{path}"
+
+  # Relative with version URL generator
   defp url(%{shop_name: domain}, path),
     do: "#{ShopifyAPI.transport()}#{domain}/admin/api/#{version()}/#{path}"
```

Comment:
Can we re-implement this version in terms of the above?

Reply:
Yup, doesn't get us much but the transport and domain don't have to be recreated.